### ISSUE TITLE
Not updating after first pick

### ISSUE THREAD

**Issue by thibaudgg** [supertriodo/Arena-Tracker#7]
Hey,

I tried a recent release (from master) and the score below the cards weren't updated after the first pick (same for the card in the app), so I pulled the latest commit from your repo and now I get the following issue when trying to build the app.

```
/Users/Thibaud/Codes/Arena-Tracker/Sources/Cards/secretcard.cpp:5: error: delegating constructors are permitted only in C++11
SecretCard::SecretCard() : SecretCard("")
                           ^~~~~~~~~~
```

So I tried to add the following to compiling with `c++11` (tried `c++14` as well):

```
QMAKE_CXXFLAGS += -std=c++14 -stdlib=libc++ -mmacosx-version-min=10.11
LIBS += -stdlib=libc++ -mmacosx-version-min=10.11
```

but I'm still getting:

```
/Users/Thibaud/Codes/Arena-Tracker/Sources/Cards/secretcard.h:23: error: only virtual member functions can be marked 'override'
    void draw() Q_DECL_OVERRIDE;
                ^~~~~~~~~~~~~~~
```

Not sure that my issue has been fixed, but it would be nice to be able to build the app again :).

**Comment by supertriodo** [comment]
I'll remove the c++ 11 code.

**Comment by supertriodo** [comment]
New release should work fine.

https://github.com/supertriodo/Arena-Tracker/archive/v3.26.zip

If Arena Tracker doesn't advance after first pick it generally means hearthstone is not writing in his log. Have you installed another tracker which messed with log.config?

**Comment by thibaudgg** [comment]
@supertriodo thanks the new release compile fine again now. But it still doesn't refresh after first pick.
I didn't installed any other tracker, my log.config (`~/Library/Preferences/Blizzard/Hearthstone/log.config`) looks like that:

```

[Bob]
LogLevel=1
ConsolePrinting=true

[Power]
LogLevel=1
ConsolePrinting=true

[Rachelle]
LogLevel=1
ConsolePrinting=true

[Zone]
LogLevel=1
ConsolePrinting=true

[Ben]
LogLevel=1
ConsolePrinting=true

[Asset]
LogLevel=1
ConsolePrinting=true

[Arena]
LogLevel=1
ConsolePrinting=true
```

Thanks!

**Comment by supertriodo** [comment]
Try playing a game in practice mode to see if the game is tracked.
If is not tracked then hearthstone is not writting in its log.

**Comment by thibaudgg** [comment]
@supertriodo nothing happens on practice mode neither, I reinstalled everything (battle.net + hearthstone) still an issue. How to check and fix log writing?

**Comment by supertriodo** [comment]
send me ArenaTrackerLog.txt inside HSCards and Player.log (Hearthstone Log) to be sure

**Comment by thibaudgg** [comment]
Sure thing:
- [ArenaTrackerLog.txt](https://github.com/supertriodo/Arena-Tracker/files/72696/ArenaTrackerLog.txt)
- [Player.log.zip](https://github.com/supertriodo/Arena-Tracker/files/72699/Player.log.zip) ( `~/Library/Logs/Unity/Player.log`)

Thanks!

**Comment by itsdarklikehell** [comment]
i too have this issue

**Comment by supertriodo** [comment]
So yes, the log is basically empty. Maybe blizzard changed the log file for Mac. I'll have to do some research.

**Comment by supertriodo** [comment]
It doesn't seem to be any change in the Blizzard side. So the log writting should work as always.
Let's try rewrite log.config

1) Open log.config and remove all it content, so let log.config as an empty file.
2) Open Arena Tracker. It will fill log.config with the correct info.
3) Close Arena Tracker and Hearthstone (If you had it opened).
4) Open Hearthstone
5) Open Arena Tracker
6) Try a match in practice mode, start the match and concide should be enough.

**Comment by thibaudgg** [comment]
@supertriodo I tried the following and still get the same issue. :'(

**Comment by thibaudgg** [comment]
My log.config looks is now:

``` 
[Bob]
LogLevel=1
ConsolePrinting=true

[Power]
LogLevel=1
ConsolePrinting=true

[Zone]
LogLevel=1
ConsolePrinting=true

[Arena]
LogLevel=1
ConsolePrinting=true
```

**Comment by supertriodo** [comment]
I didn't have too much faith on that. I still think Blizzard changed something in the mac client.

**Comment by thibaudgg** [comment]
Thanks for your help, I hope you'll find a solution. :cry:

**Comment by supertriodo** [comment]
I have decided to install an VM with Mac OS X El capitan.

1) I installed Hearthstone
2) I run Hearthstone for the 1st time and closed it.
3) I Created the log.config with the correct info in ~/Library/Preferences/Blizzard/Hearthstone/log.config
4) I run Hearthstone again, play 1 match in solo mode, concede, and close Hearthstone.
5) I checked ~/Library/Logs/Unity/Player.log and all the info is there

So it seems there is something wrong with your installation or you did something wrong, I don't know.
If you want I can send you a Player.log example containing a solo match and concede. It's a 400 KB file.
